build.bat

```diff
@@ -0,0 +1,89 @@
+:: This script will build a .NET project in Release mode, sign the generated DLLs with a provided PFX certificate,
+:: package the DLLs into a NuGet package, and sign the NuGet package with the provided PFX certificate.
+:: This script also handles pre-run cleanup (will delete old DLLs and NuGet package files)
+
+:: Requirements:
+:: - NuGet is installed on the machine and is accessible everywhere (added to PATH)
+:: - dotnet is installed on the machine and is accessible everywhere (added to PATH) (might be in C:\Program Files\dotnet)
+:: - SnInstallPfx (https://github.com/honzajscz/SnInstallPfx) is installed on the machine and is accessible everywhere (added to PATH)
+
+@ECHO OFF
+
+:: Parse command line arguments
+SET projectName=%1
+SET certFile=%2
+SET certPass=%3
+SET containerName=EasyPost
+
+:: Delete old files
+@ECHO:
+@ECHO Cleaning old files...
+@RD /S /Q lib
+DEL /S /Q /F *.nupkg
+
+:: Install certificate (needed to automate signing later on)
+@ECHO:
+@ECHO (Re-)Installing certificate to system...
+sn -d "%containerName%"
+SnInstallPfx "%certFile%" "%certPass%" "%containerName%" || GOTO :commandFailed
+
+:: Restore NuGet packages and dependencies
+@ECHO:
+@ECHO Restoring dependencies...
+:: Runs nuget restore under the hood
+:: Not really necessary, since restore is run as part of build below
+dotnet restore || GOTO :commandFailed
+
+:: Build solution in Release mode
+:: Configuration details are hard-coded
+@ECHO:
+@ECHO Building project for release...
+dotnet msbuild -property:Configuration="Release" -property:Platform="Any CPU" -target:Rebuild -restore || GOTO :commandFailed
```

Comment:
Nit: configurable items such as what release to use are better served at the top of files for visibility and also to separate potential config from logic.